### ISSUE TITLE
Feature Request: Add support for CLN v23.05

### ISSUE THREAD

**Issue by 2Fast2BCn** [jamaljsr/polar#713]
Would it be possible to add support for CLN v23.05: Austin Texas Agreement(ATXA)

**Comment by jamaljsr** [comment]
Yes, give me a day or two to add this.

**Comment by jamaljsr** [comment]
I attempted to add CLN v23.05 today but unfortunately, there are many breaking changes in this release related to renaming RPC response fields from `millisatoshi` to `msat`. Fixes need to be made in the [c-lightning-REST](https://github.com/Ride-The-Lightning/c-lightning-REST/) project which Polar uses to communicate with the node. I have started working on a PR to implement these changes but it will take some time to fully implement and get reviewed. The good thing is that I shouldn't need to release a new version of Polar once these changes are made. I can just push a new Docker image with with the updated `c-lightning-REST` and Polar cal fetch the update out of band.

I'll keep this issue open until the `c-lightning-REST` issues are resolved.
